swagger/src/main/scala/org/http4s/rho/swagger/TypeBuilder.scala

```diff
@@ -312,21 +318,26 @@ object TypeBuilder {
         if (t.typeArgs.nonEmpty) GenArray(fromType(t.typeArgs(1)))
         else GenArray()
       } else if (klass <:< typeOf[AnyVal]) {
-        fromType(
```

Comment:
When testing my fix I also discovered that `AnyValClass` breaks here. Therefore we can also use the fallback in this case as well.